### PR TITLE
TACO-596 Add runner for publisher process via Railtie

### PR DESCRIPTION
**Pull request by andyjcross**
Adds a rake task for the `PublisherProcess` as a Railtie.

### CODE REVIEW

File: lib/staged_event/configuration.rb [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+require "directive"
```
Comment (gunterja) [inline lib/staged_event/configuration.rb:3]: What is "directive" and what does it do? Is this part of Ruby or something custom?

**Review by gunterja — APPROVED** [review]
Everything looks Kosher to me. I just had that one question about the Directive module

File: lib/staged_event/configuration.rb (hunk 1 above)
Comment (andyjcross) [reply lib/staged_event/configuration.rb:3]: Part of spicerack (https://github.com/Freshly/spicerack/blob/develop/directive/lib/directive.rb)
It's for declaring the configuration (settings) available for the gem. All the stuff that goes into the initializer, basically.

Comment (gunterja) [reply lib/staged_event/configuration.rb:3]: Gracias!

**Pull request merged**